agagd/agagd_core/tables/beta.py

```diff
@@ -0,0 +1,403 @@
+# AGAGD Core Imports
+from agagd_core.models import Chapters
+from agagd_core.models import Game
+from agagd_core.models import Member
+from agagd_core.models import Tournament
+from agagd_core.models import TopDan
+from agagd_core.models import TopKyu
+from agagd_core.models import MostTournamentsPastYear
+from agagd_core.models import MostRatedGamesPastYear
```

Comment:
You could instead do something like this:

```python
from agagd.models import (
    Chapters,
    Game,
    # etc
)
```

Or,

```python
from agagd_core import models
```

and refer to the models like so: `models.Chapters`

Reply:
Probably should change all the imports: https://www.python.org/dev/peps/pep-0328/#id7.